traffic_portal/app/src/common/modules/table/serverDeliveryServices/TableServerDeliveryServicesController.js

```diff
@@ -63,7 +63,7 @@ var TableServerDeliveryServicesController = function(server, deliveryServices, $
 					return params;
 				},
 				collection: function(serverService) {
-					return serverService.getServers({ type: 'EDGE', orderby: 'hostName' });
+					return serverService.getServers({ type: server.type, orderby: 'hostName' });
```

Comment:
This doesn't filter for the server itself - meaning that if I try to clone ds assignments of server A, server A itself will appear in the dropdown.

Reply:
good point

Reply:
i'll circle back on this one after restangular is removed